### PR TITLE
Tutorial

### PR DESCRIPTION
**Pull request by ashirrwad**
Added a tutorial card that shows instructions and code for user to try on.

### CODE REVIEW

File: Backend/app.py [hunk 1]
```diff
@@ -4,7 +4,8 @@
 import docker
 import os
 import tarfile
-
+import yaml
+import json
```
Comment (everythingfunctional) [inline Backend/app.py:8]: You need to add yaml and possibly json to the Pipfile.

**Comment by everythingfunctional** [comment]
I took a look once I got it running and things look reasonable. It looks like the backend is converting the tutorial content for the frontend to use, which is reasonable, but means you probably shouldn't commit the frontend version to the repo, since it's generated automatically.

**Comment by ashirrwad** [comment]
@everythingfunctional Hi, I've added 3 more commits, tutorial now works smoothly along with some minor UI changes and a major bug fix. Please assist with review.

**Review by everythingfunctional — APPROVED** [review]
LGTM

**Pull request merged**